src/AddTodoForm.js

```diff
@@ -1,14 +1,38 @@
-import React from "react";
-
-function AddTodoForm() {
-    return (
-      <form>
-        <label htmlFor="todoTitle">Title</label>
-        <input type="text" id="todoTitle" />
-        <button type="submit">Add</button>
-      </form>
-    );
-  }
-  
+import React, { useState } from "react";
+
+function AddTodoForm({ onAddTodo }) {
+  // Create a new state variable named todoTitle with an update function named setTodoTitle
+  const [todoTitle, setTodoTitle] = useState("");
+
+  // Create a new function called handleAddTodo
+  const handleAddTodo = (event) => {
+    // Prevent the default behavior of the form submit
+    event.preventDefault();
+
+    // Invoke the onAddTodo callback prop and pass todoTitle as an argument
+    onAddTodo(todoTitle);
+
+    // Log the value of todoTitle in the console
+    console.log("Todo Title:", todoTitle);
+
+    // Reset the form so the text input value is cleared
+    setTodoTitle("");
+  };
+
+  return (
+    <form onSubmit={handleAddTodo}>
+      {/* Add a name attribute to the text input with value title */}
+      <label htmlFor="todoTitle">Title</label>
+      <input
+        type="text"
+        id="todoTitle"
+        name="title"
+        value={todoTitle}
+        onChange={(e) => setTodoTitle(e.target.value)}
+      />
```

Comment:
It's a subtle change but it would make more sense for the comment to be directly above the element that it's referencing. Also, to address some changes suggested above, you'll want to remove the `value` and `onChange` attributes

```suggestion
      <label htmlFor="todoTitle">Title</label>
      {/* Add a name attribute to the text input with value title */}
      <input
        type="text"
        id="todoTitle"
        name="title"
      />
```

Reply:
when i remove the onChange attribute, the form doesnt work